packages/ui/src/SlotTypeIcon/SlotTypeIcon.tsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+
+import { Ice, OffIce } from "@eisbuk/svg";
+
+import { SlotType } from "@eisbuk/shared";
+import { useTranslation, SlotTypeLabel } from "@eisbuk/translations";
+
+const iconLookup = {
+  [SlotType.Ice]: Ice,
+  [SlotType.OffIce]: OffIce,
+};
+
+const SlotTypeIcon: React.FC<{ type: SlotType; className?: string }> = ({
+  type,
+  className = "",
+}) => {
+  const { t } = useTranslation();
+
+  const Icon = iconLookup[type];
+
+  return (
+    <div className={[...baseClasses, className].join(" ")}>
+      <span key="icon" className="w-5 h-5 mr-2">
+        <Icon />
+      </span>
+      <span key="type">{t(SlotTypeLabel[type])}</span>
+    </div>
+  );
+};
+
+const baseClasses = [
+  "px-4",
+  "py-3",
```

Comment:
This is unnecessary (at least from the current state of the mockup)

Reply:
Plus, it's easier to position the element if there's no weird padding (for instance when using `position: absolute`, I want to specify `left: 4px` and `bottom: 4px`, the icon itself won't appear the that position, but have a padding in between)